week/templates/week/model_index_page.html

```diff
@@ -68,8 +68,8 @@ <h5 class="card-title"><b>{{ post.title|richtext }}</b></h5>
                 {% endif %}
             {% endfor %}
         </div>
-        <!--End Card Content-->  <!--adding banner space_Srishty -->
-        <div class="img-responsive " >
```

Comment:
Karthik, I see at this point you have overidden Srishty's code any specific reason for doing it, also is it affecting what Srishty has done.

Thanks

Reply:
Thanks for the clarification Karthik